jenkins-scripts/lib/generic-default-devel-windows.bat

```diff
@@ -82,7 +82,7 @@ if exist .\configure.bat (
 )
 
 :: We want to make sure that tests are enabled for this project's build
-cmake .. -DBUILD_TESTING:BOOL=%ENABLE_TESTS% || echo "Second run of cmake (for enabling the tests) failed"
+cmake .. -DBUILD_TESTING:BOOL=%ENABLE_TESTS% -DBUILDSYSTEM_TESTING:BOOL=%ENABLE_TESTS% || echo "Second run of cmake (for enabling the tests) failed"

```

Comment:
I made a change to make Windows similar to Linux about injecting in a variable the parameters needed. `add_buildsystem_testing_jrivero` branch.

I've tested it here https://build.osrfoundation.org/job/ignition_cmake-ci-pr_any-windows7-amd64/318/ and here https://build.osrfoundation.org/job/ignition_cmake-ci-master-windows7-amd64/7/.

Reply:
testing your branch here: [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign_physics-ign-2-win&build=33)](https://build.osrfoundation.org/view/ign-citadel/job/ign_physics-ign-2-win/33/) https://build.osrfoundation.org/view/ign-citadel/job/ign_physics-ign-2-win/33/

Reply:
I fast forwarded to include it here.